sentry.client.config.ts

```diff
@@ -0,0 +1,37 @@
+// This file configures the initialization of Sentry on the client.
+// The config you add here will be used whenever a users loads a page in their browser.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from '@sentry/nextjs';
+import * as Spotlight from '@spotlightjs/spotlight';
+
+Sentry.init({
+  // dsn: '___DSN___',
+
+  spotlight: process.env.NODE_ENV === 'development',
```

Comment:
I dont think its mandatory to add this boolean for nextjs

Reply:
For client config

Reply:
Thank you for your feedback, I remove the spotlight attribute in Sentry client side configuration.

Indeed, it's not needed for client config.